lib/Config.js

```diff
@@ -280,7 +280,7 @@ function dmfLocationConstraintAssert(locationObj) {
 function locationConstraintAssert(locationConstraints) {
     const supportedBackends =
           ['mem', 'file', 'scality',
-           'mongodb', 'dmf'].concat(Object.keys(validExternalBackends));
+           'mongodb', 'dmf', 'azure_archive'].concat(Object.keys(validExternalBackends));
     assert(typeof locationConstraints === 'object',
         'bad config: locationConstraints must be an object');
     Object.keys(locationConstraints).forEach(l => {
```

Comment:
Do we need to add a check like for dmf ?https://github.com/scality/cloudserver/blob/0fb54c9d3153dded1a0f45d144e1d8448eda7e75/lib/Config.js#L391-L393

Reply:
even though it should not really be relevant (since cloudserver is not connecting to this kind of location), for consistency i think it would be better indeed...

Reply:
Done [bc20467](https://github.com/scality/cloudserver/pull/5008/commits/bc20467fcc26794be0b5eeb153d0c4ab5b31c3ea)